### PR TITLE
Add March Meetup post

### PR DESCRIPTION
**Pull request by matixmatix**

### CODE REVIEW

File: _posts/2017-03-30-march-meetup-recap.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+author: jurgen
+bg_photo: /assets/img/posts/2017-03-30-meetup-cover.png
+---
+
+Yet again, two amazing talks unfolded right in front of our eyes in [Poligon](http://www.poligon.si/).
+
+The [first one](https://docs.google.com/presentation/d/1LVSgfWiceIqAMiC0qTpaxc2sEkBkxwtT5zx6C8k35ss/pub?start=false&loop=false&delayms=60000) was about [Rack](http://rack.github.io/). [Krištof Črnivec](https://www.facebook.com/kristof.crnivec) reminded us about this essential gem, emphasizing how it powers the most of Ruby web servers in the wild. He showed us some examples of its clear and well-thought-out interface.  
+
+Second talk was delivered by [Roman Orač](https://github.com/romanorac). The main topic was about machine learning, and how they're applying it at [1Tap Receipts](http://1tapreceipts.com/) to categorize photographs of receipts. Some real next-level stuff was revealed here, followed by an intensive Q&A session, which confirmed the interest for machine learning being at an all time high.
+
```
Comment (otobrglez) [inline _posts/2017-03-30-march-meetup-recap.md]: @matixmatix please embed this:

```html
<iframe src="//www.slideshare.net/slideshow/embed_code/key/bdJXYour2AqiF" width="595" height="485" frameborder="0" marginwidth="0" marginheight="0" scrolling="no" style="border:1px solid #CCC; border-width:1px; margin-bottom:5px; max-width: 100%;" allowfullscreen> </iframe> <div style="margin-bottom:5px"> <strong> <a href="//www.slideshare.net/RomanOra/gentle-introduction-to-machine-learning" title="Gentle introduction to Machine Learning" target="_blank">Gentle introduction to Machine Learning</a> </strong> from <strong><a target="_blank" href="//www.slideshare.net/RomanOra">Roman Orač</a></strong> </div>
```

File: _posts/2017-03-30-march-meetup-recap.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+layout: post
+title: Ruby on Rack and Machine Learning / March Meetup Recap
+author: jurgen
+bg_photo: /assets/img/posts/2017-03-30-meetup-cover.png
+---
+
+Yet again, two amazing talks unfolded right in front of our eyes in [Poligon](http://www.poligon.si/).
+
+The [first one](https://docs.google.com/presentation/d/1LVSgfWiceIqAMiC0qTpaxc2sEkBkxwtT5zx6C8k35ss/pub?start=false&loop=false&delayms=60000) was about [Rack](http://rack.github.io/). [Krištof Črnivec](https://www.facebook.com/kristof.crnivec) reminded us about this essential gem, emphasizing how it powers the most of Ruby web servers in the wild. He showed us some examples of its clear and well-thought-out interface.  
+
```
Comment (otobrglez) [inline _posts/2017-03-30-march-meetup-recap.md]: @matixmatix Please embed this here:

```html
<iframe src="https://docs.google.com/presentation/d/1LVSgfWiceIqAMiC0qTpaxc2sEkBkxwtT5zx6C8k35ss/embed?start=false&loop=false&delayms=60000" frameborder="0" width="1058" height="823" allowfullscreen="true" mozallowfullscreen="true" webkitallowfullscreen="true"></iframe>
```

**Comment by matixmatix** [comment]
@otobrglez slide and video embeds are now added

**Comment by otobrglez** [comment]
Tnx @matixmatix

**Comment by mrfoto** [comment]
🎉

File: _posts/2017-03-30-march-meetup-recap.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+<iframe src="//www.slideshare.net/slideshow/embed_code/key/bdJXYour2AqiF"  frameborder="0" marginwidth="0" marginheight="0" scrolling="no" height="400" style="border:1px solid #CCC; border-width:1px; margin-bottom:5px; max-width: 100%; width: 100%;" allowfullscreen> </iframe> 
+
+<iframe height="400" src="https://www.youtube.com/embed/zKnHXR_pnRA" frameborder="0" allowfullscreen style="width:100%"></iframe>
+
+We are thankful for [Toptal](https://www.toptal.com/) providing us 🍻 and for [1Tap Receipts](http://1tapreceipts.com/) bringing us 🍕.
+
+<div class="gallery">
+  <a href="/assets/img/posts/2017-03-30-meetup-2.jpg" target="_blank" style="display: block; width: 100%">
+    <img src="/assets/img/posts/2017-03-30-meetup-2.jpg" alt="Ruby meetup - February 2017" style="width: 100%">
```
Comment (mrfoto) [inline _posts/2017-03-30-march-meetup-recap.md]: Tale alt si pozabu spremenit, ko si kopi pejstu 😄

**Pull request merged**